sphinx/ext/autosummary/generate.py

```diff
@@ -201,6 +224,21 @@ def get_members(obj, types, include_public=[], imported=True):
                           if x in include_public or not x.startswith('_')]
                 return public, items
 
+            def get_modules(obj, include_public=[]):
+                # type: (Any, List[str]) -> Tuple[List[str], List[str]]
+                items = []  # type: List[str]
+                for _, modname, ispkg in pkgutil.iter_modules(obj.__path__):
+                    fullname = name + '.' + modname
+                    try:
+                        import_by_name(fullname)
```

Comment:
Why do you check the module is importable here? No check is needed here to me.

Reply:
Removed it